workspaces/nexus-repository-manager/plugins/nexus-repository-manager/app-config.example.yaml

```diff
@@ -0,0 +1,14 @@
+proxy:
+  endpoints:
+    '/nexus-repository-manager':
+      target: ${NEXUS_REPOSITORY_MANAGER_URL}
+      headers:
+        X-Requested-With: 'XMLHttpRequest'
+        # Uncomment the following line to access a private Nexus Repository Manager using a token
+        # Authorization: 'Bearer <NEXUS_REPOSITORY_MANAGER_TOKEN>'
```

Comment:
I renamed this from `app-config.yaml` to `app-config.example.yaml` because it contains no authorization by default.